packages/git-extractor/src/routes/github/api.ts

```diff
@@ -170,7 +171,7 @@ export async function getRepo(username: string, repo: string, token: string) {
 
   const response: { data: IRepoResponse } = await axios({
     url,
```

Comment:
Might be good to add our default credentials if the token is not available (to not stumble upon rate limiting) by adding https://github.com/codesandbox/codesandbox-importers/blob/a7d9f75365cfe744fbf02f7c5bd1b02677ebce15/packages/git-extractor/src/routes/github/api.ts#L48 to the URL if no token is found.